tests/topotests/bgp_snmp_bgp4v2_notification/test_bgp_snmp_bgp4v2_notification.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
```

Comment:
Please add SPDX-License tag.

Reply:
Hello @ton31337 
Thank you for your comments.


SPDX-License tag fixed and configs are reduced.